ingestion/setup.py

```diff
@@ -275,6 +275,7 @@
     # For publishing
     "twine",
     "build",
+    plugins["sample-data"],
```

Comment:
does this work or we need to unpack something?

Reply:
tested it out, `*plugins["sample-data"]` worked as expected